src/SurveyService.Models.Dto/Requests/Question/CreateQuestionRequest.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+
+namespace LT.DigitalOffice.SurveyService.Models.Dto.Requests.Question;
+
+public record CreateQuestionRequest
+{
+  public string Content { get; set; }
+  public DateTime? Deadline { get; set; }
+  public bool HasRealTimeResult { get; set; }
+  public bool IsAnonymous { get; set; }
+  public bool IsRevoteAvailable { get; set; }
+  public bool IsObligatory { get; set; }
+  public bool IsPrivate { get; set; }
+  public bool HasMultipleChoice { get; set; }
+  public bool HasCustomOptions { get; set; }
+  public bool IsActive { get; set; }
```

Comment:
same as group

Reply:
I'd better just remove CreateQuetionRequest, this model someone else's responsibility.